phpBB/composer.json

```diff
@@ -25,7 +25,7 @@
 		"phpbb/phpbb-core": "self.version"
 	},
 	"require": {
-		"php": ">=5.4,<7.1",
+		"php": ">=5.4",
```

Comment:
No, I actually did that on purpose. Previously we had to update this requirement every time there is a new version. Instead of explicitly stating no compatibility with newer versions (which we don't actually know), we now implicitly state support unless we discover something else (in which case we might just have to change a few lines of code).